var/spack/repos/builtin/packages/hacckernels/package.py

```diff
@@ -33,7 +33,7 @@ class Hacckernels(CMakePackage):
     cores and beyond."""
 
     homepage = "https://xgitlab.cels.anl.gov/hacc/HACCKernels"
-    url      = "https://xgitlab.cels.anl.gov/hacc/HACCKernels.git"
+    # url      = "https://xgitlab.cels.anl.gov/api/v4/projects/hacc%2FHACCKernels/repository/archive.tar.gz"
```

Comment:
A lot of things you wouldn't expect break in Spack when packages don't have any `url`. I would uncomment this.

Reply:
Case in point:
```console
$ spack versions hacckernels
==> Error: <class 'spack.pkg.builtin.hacckernels.Hacckernels'>
```